BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/BrainCloudComms.cs

```diff
@@ -2100,7 +2156,65 @@ private static long GetJsonLong(Dictionary<string, object> jsonData, string key,
 
             return defaultReturn;
         }
+        /// <summary>
+        /// Attempts to create and send next request bundle.
+        /// If to many attempts have been made, the request becomes an error
+        /// </summary>
+        /// <param name="status">Current Request Status</param>
+        /// <param name="bypassTimeout">Was there an error on the request?</param>
+        private void RetryRequest(RequestState.eWebRequestStatus status, bool bypassTimeout)
```

Comment:
Wait, the retry already worked in C#, is this a new method? i don't see the removal of other retry code.

Reply:
There is a retry method within C#, I wanted to retry in another spot so I made this into a function. I forgot however to delete the method and replace it with the function to show where it came from.